internal/controllers/port/actuator.go

```diff
@@ -77,7 +75,7 @@ func (actuator portActuator) ListOSResourcesForAdoption(ctx context.Context, obj
 	return actuator.osClient.ListPort(ctx, listOpts), true
 }
 
-func (actuator portCreateActuator) ListOSResourcesForImport(ctx context.Context, filter orcv1alpha1.PortFilter) portIterator {
+func (actuator portActuator) ListOSResourcesForImport(ctx context.Context, filter orcv1alpha1.PortFilter) portIterator {
```

Comment:
FYI, this is going to conflict with https://github.com/k-orc/openstack-resource-controller/pull/302. Depending on which one merges first, you might need to update your PR.

Reply:
fine, I'll let you merge your pr first if you like, then you can rebase this one otherwise I'll do it.